OOP/OOP.cs

```diff
@@ -0,0 +1,267 @@
+            }
+        }
+    }
+}
+
+namespace OOP
+{
+    public class Classroom : IPrintable
+    {
+        private List<Person> _persons = new();
+
+        public void AddPersons()
+        {
+            var person1 = new Person("Pupkin", "Vasiliy", 20);
+            var person2 = new Person("Kirkorov", "Petya", 46);
+
+            _persons.Add(person1);
+            _persons.Add(person2);
+        }
+
+        public void RemovePersons()
+        {
+            _persons.RemoveAll(x => x == _persons[0]);
+        }
+
+        public void ClearPersons()
+        {
+            _persons.Clear();
+        }
+
+        public void Print()
+        {
+            foreach (var person in _persons)
+            {
+                string output = string.Join(" ", person.ToString());
+                Console.WriteLine($"{output}");
+            }
+        }
+    }
+}
+
+namespace OOP
+{
+    public class Person : IPrintable
+    {
+        private string _personSurname;
+        private string _personName;
+        private int _personAge;
+        public string PersonSurname => _personSurname;
+
+        public string PersonName => _personName;
+
+        public int PersonAge => _personAge;
+
+        public Person(string personSurname, string personName, int personAge)
+        {
+            _personSurname = personSurname;
+            _personName = personName;
+            _personAge = personAge;
+        }
+
+        public override string ToString()
+        {
+            return $"{PersonSurname} {PersonName}, {PersonAge}";
+        }
+
+        public void SetSurname(string personSurname)
+        {
+            _personSurname = personSurname;
+        }
+
+        public void SetName(string personName)
+        {
+            _personName = personName;
+        }
+
+        public void SetAge(int personAge)
+        {
+            _personAge = personAge;
+        }
+
+        public void WritePersonData()
+        {
+            Console.WriteLine($"{PersonSurname} {PersonName}, {PersonAge}");
+        }
+
+        public void Print()
+        {
+            Console.WriteLine($"{PersonSurname} {PersonName}, {PersonAge} y.o.");
+        }
+
+        public string GetSurname()
+        {
+            return _personSurname;
+        }
+
+        public string GetName()
+        {
+            return _personName;
+        }
+
+        public int GetAge()
+        {
+            return _personAge;
+        }
+    }
+}
+
+namespace OOP
+{
+    public class Student : Person, IPrintable
+    {
+        private string _studentAbility;
+        private int _studentAssesment;
+
+        public Student(string personSurname, string personName, int personAge) : base(personSurname, personName, personAge)
+        {
+            Surname = personSurname;
+            Name = personName;
+            Age = personAge;
+        }
+
+        public string StudentAbility => _studentAbility;
+
+        public int StudentAssesment => _studentAssesment;
```

Comment:
Maintain a uniform style of order in the classroom.
Or everywhere the properties are higher than the constructor.
Or everywhere below.
As I write above

Reply:
not fixed